[FIX] Not enough argument on Controller@getContentBasedOnType.

krisanalfa opened the pull request:
Fixes #2724

krisanalfa committed:
[FIX] Not enough argument on Controller@getContentBasedOnType.

krisanalfa committed:
[FIX] Remove unused imports.

fletch3555 commented:
I think a better fix would be to add `null` as a default value in the [function definition](https://github.com/the-control-group/voyager/blob/1.x-dev/src/Http/Controllers/Controller.php#L177), so it's optional.

Thanks for including the test, also!

krisanalfa committed:
[UPDATE] Change Controller@getContentBasedOnType signature.

- Fourth argument, `options` is optional, with default = `null`.
- Add a little more testing for more reliability.

krisanalfa committed:
[UPDATE] Code style.

krisanalfa commented:
Hi, @fletch3555, thanks for your review. I've updated my PR. Kindly please check. Thanks.
